src/inverseRobot.js

```diff
@@ -8,6 +8,21 @@
 
 function inverseRobot(robot) {
   // write code here
+  const valueArr = Object.values(robot);
+
+  for (let i = 0; i < valueArr.length; i++) {
+    if (valueArr.indexOf(valueArr[i]) !== valueArr.lastIndexOf(valueArr[i])) {
+      return null;
+    }
+  }
```

Comment:
Maybe, its better to use hasOwnProperty()

Reply:
Yes, it seems quite better than my solution. Thanks!